src/redux/Categories/categoriesSlice.js

```diff
@@ -0,0 +1,15 @@
+import createSlice from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: {
+    list: [],
+    status: 'Under construction',
+  },
+  reducers: {
+    setStatus: (state) => {
+      state.status = 'Under construction';
+    },
+  },
+});
+export default categoriesSlice;
```

Comment:
- [ ] it would be better to export `setStatus`  as an action in order to complete the requirements 
example : `export const { setStatus } = categoriesSlice.actions`